docs/source/_templates/layout.html

```diff
@@ -3,3 +3,24 @@
 {# Silence the navbar #}
 {% block docs_navbar %}
 {% endblock %}
+
+<!--
+    Custom footer
+-->
+{% block footer %}
+<!-- Based on pydata_sphinx_theme/footer.html -->
```

Comment:
I am not a sphynx expert by any stretch of the imagination -- if there is a better way to do this I would appreciate hearing about it